src/pe_source/data/sixgill/config.py

```diff
@@ -0,0 +1,45 @@
+"""Get cybersixgill API token."""
+
+# Standard Python Libraries
+from configparser import ConfigParser
+import glob
+import os
+
+# Third-Party Libraries
+import requests
+
+# Configuration
+SECTION = "sixgill"
+BASE_DIR = os.path.abspath(os.path.join(__file__, "../../../.."))
+REPORT_DB_CONFIG = glob.glob(f"{BASE_DIR}/**/*.ini", recursive=True)[0]
+
+
+def token():
+    """Retrieve bearer token from Cybersixgill client."""
+    if os.path.isfile(REPORT_DB_CONFIG):
+        parser = ConfigParser()
+        parser.read(REPORT_DB_CONFIG, encoding="utf-8")
+        if parser.has_section(SECTION):
+            params = parser.items(SECTION)
+            _id, _secret = params[0], params[1]
+            client_id = _id[1]
+            client_secret = _secret[1]
+        else:
+            raise Exception(
+                "Section {} not found in the {} file".format(SECTION, REPORT_DB_CONFIG)
+            )
+    else:
+        raise Exception("Database.ini file not found.")
```

Comment:
It might be nice to output the path that you're looking at for this file.

Reply:
Good idea. Added here: https://github.com/cisagov/pe-reports/pull/117/commits/cb34664c54753f99f68446f1ea08b2feef98c364